querydsl-sql/src/test/java/com/querydsl/sql/SubqueriesBase.java

```diff
@@ -15,7 +29,11 @@
 import com.google.common.collect.ImmutableList;
 import com.querydsl.core.testutil.ExcludeIn;
 import com.querydsl.core.types.SubQueryExpression;
-import com.querydsl.core.types.dsl.*;
+import com.querydsl.core.types.dsl.Expressions;
+import com.querydsl.core.types.dsl.NumberPath;
+import com.querydsl.core.types.dsl.Param;
+import com.querydsl.core.types.dsl.PathBuilder;
```

Comment:
So sorry, I have bad news.
I deleted my forked querydsl repo yesterday by misunderstood and refork it.
But I still had local repository.
What is the best way that I had to do for your requirement?